tests/v2/validation/rbac/globalrolesv2/README.md

```diff
@@ -9,11 +9,10 @@ Global Roles v2 introduces enhanced capabilities, allowing users to define permi
 Your GO suite should be set to `-run ^TestGlobalRolesV2TestSuite$`. You can find specific tests by checking the test file you plan to run.
 
 In your config file, set the following:
-```
-rancher: 
+```json
```

Comment:
Anu, this is not in json format. We can remove this

Reply:
Changed it to yaml, thanks for pointing that out.